chisel/blink/Makefile

```diff
@@ -31,6 +31,7 @@ blink.json: Blink.sv
 blink.asc: blink.json
 	nextpnr-ice40 \
 		$(PNRFLAGS) \
+		--pcf ../../pcf/fomu-$(FOMU_REV).pcf \
```

Comment:
In `board.mk` variable `PCF` is set. Therefore, this might be `--pcf $(PCF)` as in the hdl examples: https://github.com/im-tomu/fomu-workshop/blob/d72b7f0b790ce0837f7ae475443e653c2339d5df/hdl/PnR_Prog.mk#L8.

Reply:
I tried that but `PCF` contains one to many `..`, so that it ends up looking in the wrong location. We could either add a fake directory at the beginning, make `PCF` contain an absolute path or move the chisel directory.

Reply:
That's correct. Thanks for pointing it out. `board.mk` has PCF_PATH, so it should be possible to set it in `chisel/blink/Makefile`, before including the makefile:

```
PCF_PATH = ../../pcf
include ../../hdl/board.mk

...
```

Reply:
That's pretty cool, I didn't know about the ?= construct